### PR TITLE
fix: Adds missing words to `companies`

### PR DESCRIPTION
**Pull request by anthony0030**
# Fix Dictionary

Dictionary: ```companies```

## Description
Adds missing words to companies
* viber
* waze
* imgix
* moovit
* cdnjs
* hulu
* papaki
* imessage

Fixes word order
## References

https://github.com/streetsidesoftware/cspell-dicts/issues/1897

## Checklist

- [X] By submitting this issue, you agree to follow our
      [Code of Conduct](https://github.com/streetsidesoftware/cspell-dicts/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: dictionaries/companies/src/companies.txt [hunk 1]
```diff
@@ -774,6 +776,8 @@ Illinois Tool Works Inc.
 Illumina (company)
 Illumina Inc
 ILMN
+imessage
```
Comment (Jason3S) [inline dictionaries/companies/src/companies.txt:779]: Is `imessage` a company?

Comment (Jason3S) [reply dictionaries/companies/src/companies.txt:779]: I think it is better to move it to [cspell-dicts/software-services.txt](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/src/software-services.txt)

**Comment by Jason3S** [comment]
@anthony0030,

The file was already sorted (most likely with a different locale than you are using).

File: dictionaries/companies/src/companies.txt (hunk 1 above)
Comment (anthony0030) [reply dictionaries/companies/src/companies.txt]: Good point.
I added it in https://github.com/streetsidesoftware/cspell-dicts/pull/1941

**Comment by Jason3S** [comment]
@anthony0030,

I'm adding a script that will auto sort the words after a PR has landed. So no need to find the right location.

**Comment by anthony0030** [comment]
So if I understand correctly, if I had just added everything to the bottom or top of the file your script would have sorted everything automatically.

Also thank you @Jason3S for the merge!

**Comment by Jason3S** [comment]
@anthony0030,

You are not the first person to run into this issue. But, you did prompt me to write a script that will just auto lint the source dictionary files. It is a pain to be forced to add them in the right place.

**Pull request merged**